src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -30,6 +33,20 @@ public interface IexClient {
    * @return a list of the last traded price for each of the symbols passed in.
    */
   @GetMapping("/tops/last")
-  List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
+  List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols,
+      @RequestParam ("token") String token);
+
+  /**
+   * Get the last traded price for each stock symbol passed in.
+   *
+   * @param from date that the prices start from.
+   * @return a list of the last traded price for each of the symbols passed in.
+   */
+
+  @GetMapping("/data/core/historical_prices/{symbols}")
+  List<IexHistoricalPrices> getHistoricalPricesFrom(@RequestParam("from") Date from,
+      @PathVariable("symbols")String symbols,
```

Comment:
Why is symbols a path variable?

Reply:
IEX has it as a path parameter and not a query parameter. For some reason it's just there for historical prices and not last trade price. I haven't been able to find a different way to do it but if you have any suggestions I can look into changing it.